dGame/dInventory/Item.cpp

```diff
@@ -344,6 +344,11 @@ void Item::DisassembleModel() {
 	std::string lxfmlPath = "BrickModels/" + GeneralUtils::SplitString(renderAssetSplit.back(), '.').at(0) + ".lxfml";
 	auto buffer = Game::assetManager->GetFileAsBuffer(lxfmlPath.c_str());
 
+	if (!buffer.m_Success) {
+		Game::logger->Log("Item", "Failed to load %s", lxfmlPath.c_str());
```

Comment:
I'd like to imagine we can be more descriptive of this error here and explain _why_ this fail was encountered